streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -98,8 +97,9 @@
 
     private StreamPartitionAssignor partitionAssignor = null;
 
-    private long lastClean;
-    private long lastCommit;
+    private long currTimeMs;
```

Comment:
I'm not sure why this needs to be a field. It looks like it is used to measure latency, but then why not just use a local variable around the code that is being timed?, i.e, 
```
final long start = time.milliseconds()
// do some computations
computeLatency(start)
```
Also, the name doesn't seem correct as it isn't really the current time. It is more like `timerStartedMs`